### PR TITLE
Add unit tests to CI for pull-gcp-compute-persistent-disk-csi-driver

### PR DESCRIPTION
**Pull request by davidz627**
/assign @krzyzacy

### CODE REVIEW

**Comment by krzyzacy** [comment]
wanna put it under https://github.com/kubernetes/test-infra/tree/master/config/jobs with the other job, and add yourself as an OWNER? :-)

**Comment by davidz627** [comment]
@krzyzacy moved to its own folder. PTAL.

File: config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/config.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+        args:
+        - "--repo=sigs.k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--clean"
+        - "--timeout=300"
```
Comment (krzyzacy) [inline config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/config.yaml]: add a `--scenario=execute` here, and below

**Pull request merged**